lib/configuration.js

```diff
@@ -149,16 +145,20 @@ function gatherConfiguration(initConfiguration) {
         'An invalid configuration was given to the Stackdriver Error Library');
   }
 
+  if (!isString(projectId)) {
+     if (isString(env.GCLOUD_PROJECT) || isNumber(env.GCLOUD_PROJECT)) {
+
+      projectId = env.GCLOUD_PROJECT;
```

Comment:
This doesn't seem logically related to uncaught exceptions. Can this be moved to a PR of its own?